src/ape_accounts/_cli.py

```diff
@@ -5,13 +5,14 @@
 from eth_utils import to_bytes
 
 from ape import accounts
+from ape._cli import ApeCLI
 from ape.utils import notify
 
 # NOTE: Must used the instantiated version of `AccountsContainer` in `accounts`
 container = accounts.containers["accounts"]
 
 
-@click.group(short_help="Manage local accounts")
+@click.group(cls=ApeCLI, name="accounts", short_help="Manage local accounts")
```

Comment:
Is `cls=ApeCLI` supposed to be there?

Reply:
Yess

Reply:
I made `ApeCLI` a different type than it was before

Reply:
Should it be with all the rest of them then?

Reply:
Oh oops, you are right, this shouldn't be here